lang/en/left-menu.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+return [
+	/*
+	|--------------------------------------------------------------------------
+	| Left Menu
+	|--------------------------------------------------------------------------
+	*/
+	'back_to_gallery' => 'Back to Gallery',
+
+	'admin' => 'Admin',
+	'clockwork' => 'Clockwork App',
+	'logs' => 'Show Logs',
+	'jobs' => 'Show job history',
+	'user' => 'User',
+
+	'sign_out' => 'Sign out',
+
+	'about' => 'About',
+	'api' => 'API documentation',
+	'source_code' => 'Source code',
```

Comment:
```suggestion
	'source_code' => 'Source Code',
```
Just keeping the cases consistent in the menu